app/Services/StoreFileService.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Services;
+
+use Illuminate\Http\Request;
+
+class StoreFileService
+{
+    protected Object $file;
```

Comment:
Probably we can use `Illuminate\Http\UploadedFile` type here to be more specific
```suggestion
    protected UploadedFile $file;
```